src/styles/index.ts

```diff
@@ -3772,6 +3772,14 @@ const styles = (theme: ThemeColors) =>
             alignItems: 'center',
         },
 
+        bankAccountMenuItem: {
+            paddingLeft: 0,
+            paddingRight: 0,
+            borderRadius: variables.buttonBorderRadius,
```

Comment:
Why do we need border radius here?

Reply:
@daledah Did you have a chance to look at the question above? Thanks.